### ISSUE TITLE
Can't find the file 'glove.6B.300d.model.bin'

### ISSUE THREAD

**Issue by zhouzhouyang520** [chengjl19/PAL#4]
Hello, I am using your project. I found that I could not find the glove.6B.300d.model.bin file when evaluating. Could you please provide that file? Thank you.

**Comment by chengjl19** [comment]
You could download it from here: https://nlp.stanford.edu/projects/glove/

**Comment by zhouzhouyang520** [comment]
Thank you very much for your prompt feedback. I have downloaded the file 'glove.6B.300d.txt', and tried pre-processing this file. During the process, I found that the format of the file 'glove.6B.300d.model.bin' is different from the format of 'glove.6B.300d.txt', which leads to errors. May I ask if the bin file only contains vectors without the first column, i.e. words?

**Comment by chengjl19** [comment]
Maybe you can follow this https://stackoverflow.com/questions/60055813/understanding-usage-of-glove-vectors, and set binary=True

**Comment by zhouzhouyang520** [comment]
Thank you very much.
I have solved this issue following this step:

1. Download the file `glove.6B.300d.txt` from  https://nlp.stanford.edu/projects/glove/

2. Then, I converted the file using the following code: python -m gensim.scripts.glove2word2vec --input=glove.6B.300d.txt --output=glove.6B.300d.model.bin

3. Next, I changed the code `self.m = KeyedVectors.load(os.path.join(path, 'glove.6B.300d.model.bin'), mmap='r')` to `self.m = KeyedVectors.load_word2vec_format(os.path.join(path, 'glove.6B.300d.txt'), binary=False, encoding='utf8')`.

I have solved this problem, thank you!
